alchemist/alchemist-incarnation-biochemistry/src/test/resources/provaBCReaction.yml

```diff
@@ -0,0 +1,24 @@
+incarnation: biochemistry
```

Comment:
File names must be in English (testBCReaction). This also happens in another file.